indico/modules/events/registration/templates/management/regform.html

```diff
@@ -1,6 +1,32 @@
 {% extends 'events/registration/management/_regform_base.html' %}
 
 {% block content %}
+    {% block regform_private %}
```

Comment:
For what kind of override do you plan to use this block?

Reply:
Just there if ever there is a need to change the text, title, style...
We don't need now but in case later on, to be more flexible.